src/uenv/log.cpp

```diff
@@ -0,0 +1,21 @@
+#include <memory>
+
+#include "spdlog/sinks/stdout_color_sinks.h"
+#include "spdlog/sinks/syslog_sink.h"
+#include <spdlog/spdlog.h>
+
+#include "log.h"
+
+namespace uenv {
+void init_log() {
+    auto console_sink = std::make_shared<spdlog::sinks::stdout_color_sink_mt>();
```

Comment:
To do: don't use `_mt` (multithreaded) version?

Reply:
Changed in https://github.com/bcumming/uenv2/pull/3/commits/20caa3a239c5bf6f8ee4971c626bb025b4856df4. If you're ever planning to do any logging from multiple threads this is important to be aware of.

We could also just use the multi-threaded loggers as it's unlikely to have any sort of measurable impact. What do you think?